tests/accessor/local_accessor_access_among_work_items_fp64.cpp

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for local_accessor.
+//
+//  This test provides verifications that local_accessor can access the memory
+//  shared among work-items.
```

Comment:
+ "in the case of fp64 accesses when available"

Reply:
I suggest add `For the double type.` instead. Sentences like this uses across all tests.
I've done it [here](https://github.com/KhronosGroup/SYCL-CTS/pull/364/commits/a9ad5ace687645d284a860d46e9036888371a92c#diff-336d0aecd22b6ac2375b4d894a47e5370c5e7553b1646c1a2d488cf7eb64126bR8)